pkgs/applications/misc/sampler/default.nix

```diff
@@ -11,7 +11,15 @@ buildGoModule rec {
     sha256 = "1lanighxhnn28dfzils7i55zgxbw2abd6y723mq7x9wg1aa2bd0z";
   };
 
-  vendorSha256 = "04nywhkil5xkipcibrp6vi63rfcvqgv7yxbxmmrhqys2cdxfvazv";
+  patches = [
```

Comment:
we should wait for a release or use an untagged commit - randomly pulling in patches sucks for everyone.

Reply:
There isn't one, yet. We can wait, but then we'll need to keep this EOL go release around.